rma/i18n/es.po

```diff
@@ -0,0 +1,16 @@
+# Translation of Odoo Server.
```

Comment:
2015-08-05 15:48 GMT-05:00 Pedro M. Baeza <notifications@github.com>:

> I think you don't need these files as this module doesn't have any
> translation


The same than before, we use them in production with a "NO WARNING RULE"
and this company use 3 spanish from time to time they will have little
diferences but for now the empty file save us the warning of not transated
elements.

It save us a little problems of have false warnings, just warnings that
muct be taken into account.

Do you think you can accept such situation?


-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>